super-csv/src/main/java/org/supercsv/prefs/CsvPreference.java

```diff
@@ -430,21 +448,34 @@ public Builder maxLinesPerRow(final int maxLinesPerRow) {
 		}
 		
 		/**
-		 * Uses an EmptyColumnParsing to determine whether empty String (i.e. "") should be read as empty string instead as null. 
+		 * Uses an EmptyColumnParsing to determine whether empty String (i.e. "") should be read as empty string instead as null.
 		 * The default is <tt>ParseEmptyColumnsAsNull</tt>.
-		 * 
+		 *
 		 * @since 2.4.1
 		 * @param emptyColumnParsing
 		 *            the emptyColumnParsing
 		 * @return the updated Builder
-		 */		
+		 */
 		public Builder setEmptyColumnParsing(final EmptyColumnParsing emptyColumnParsing) {
 			if( emptyColumnParsing == null ) {
 				throw new NullPointerException("emptyColumnParsing should not be null");
 			}
 			this.emptyColumnParsing = emptyColumnParsing;
 			return this;
 		}
+
+		/**
+		 * Value indicating the character to use for escaping a quote char.
```

Comment:
Change to: Value indicating the character to use for escaping a quote char. The default  value of QuoteEscapeChar is "

or something..